src/accessControl/acp/index.js

```diff
@@ -201,18 +201,35 @@ function ensureAccessControl(policyUrl, datasetWithAcr, changed) {
 }
 
 function ensureApplyControl(policyUrl, datasetWithAcr, changed) {
-  const accessControls = acp.getAllControl(datasetWithAcr);
+  let accessControls = [];
+  try {
+    accessControls = acp.getAllControl(datasetWithAcr);
+  } catch (error) {
+    // TODO: Handle this error
+    // It doesn't seem to affect the outcome though, so we'll leave it like this for now
```

Comment:
Yikes. What are the possible errors here?

Reply:
Good question - It seems that the `acp.getAllControl` tries to to a `getThingAll` on the dataset, but for some reason it fails. I couldn't find any good solution, and I think we probably want to rework this anyway when we get the high level API for the Solid client, so thought it wasn't worth it to debug deeper.

Reply:
Can you update the comment with this explanation?